source/tutorial/mongoid-documents.txt

```diff
@@ -379,7 +379,7 @@ for use in criteria. For example we may want to write a query like so:
   point = Point.new(12, 24)
   Venue.where(location: point)
 
-Not that when accessing custom fields from the document, you will get a new instance
+Note that when accessing custom fields from the document, you will get a new instance
```

Comment:
yeah, it should be 

"This is because Mongoid is generating a new object from the raw attributes on each access."